addons/docs/src/frameworks/common/enhanceArgTypes.test.ts

```diff
@@ -0,0 +1,102 @@
+import yaml from 'yaml';
+import { ArgType } from '@storybook/api';
+import { enhanceArgTypes } from './enhanceArgTypes';
+
+expect.addSnapshotSerializer({
+  print: (val: any) => yaml.stringify(val).trim(),
```

Comment:
I'm not sure if you changed this already but the mental translation of objects that are sometimed mapped directly through from json => yaml seems unnecessary.

Reply:
Fixed